app/controllers/home_controller.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class HomeController < ApplicationController
+  # GET /home/index
+  def index
+    respond_to do |format|
+      format.html { render 'home/index' }
+    end
+  end
+
+  # GET /home/get_companies
+  def show
+    @companies = User.unscoped.select('companies.name, companies.subdomain').where(email: params[:email]).joins(:company)
+    respond_to do |format|
+      format.html
+      format.json { render json: { companies: @companies } }
```

Comment:
json format is being used any where?

Reply:
i did it for the sake of understanding, when i put .json at the end of the link it shoes json data instead of raising error